Parsers/ASimAuditEvent/Parsers/ASimAuditEventSentinelOne.yaml

```diff
@@ -0,0 +1,677 @@
+                                        "Low",
+                                        "Informational"
+                                    );
+        let ParsedActivitydata = union
+                groupsiteactivitydata,
+                machineactivitydata,
+                accountactivitydata,
+                useractivitydata,
+                rawotheractivitydata,
+                parsedotheractivitydata_eventtype,
+                parsedotheractivitydata_objectvalue,
+                parsedotheractivitydata_severity
+            | lookup EventSeverityLookup on EventResult
+            | lookup EventSeverityLookup_activity on activityType_d;
+        let UnParsedActivitydatawithThreat = ParsedActivitydata
+            | where isnotempty(threatId_s)
+            | lookup Threatsdata on $left.threatId_s == $right.threatInfo_threatId_s
+            | project-reorder
+                threatId_s,
+                activityType_d,
+                primaryDescription_s,
+                TimeGenerated,
+                TimeGenerated1,
+                threatInfo_identifiedAt_t,
+                threatInfo_updatedAt_t
+            | where TimeGenerated1 >= TimeGenerated
+            | summarize arg_min(TimeGenerated1, *) by activityType_d, threatId_s, createdAt_t, TimeGenerated;
+        let undefineddata = UnParsedActivitydatawithThreat
+            | where threatInfo_confidenceLevel_s == "Undefined"
+            | lookup ThreatConfidenceLookup_undefined on threatInfo_analystVerdict_s;
+        let suspiciousdata = UnParsedActivitydatawithThreat
+            | where threatInfo_confidenceLevel_s == "suspicious"
+            | lookup ThreatConfidenceLookup_suspicious on threatInfo_analystVerdict_s;
+        let maliciousdata = UnParsedActivitydatawithThreat
+            | where threatInfo_confidenceLevel_s == "malicious"
+            | lookup ThreatConfidenceLookup_malicious on threatInfo_analystVerdict_s;
+        let ParsedActivitydatawithThreat = union undefineddata, suspiciousdata, maliciousdata
+            | extend
+                ThreatConfidence = coalesce(ThreatConfidence_undefined, ThreatConfidence_suspicious, ThreatConfidence_malicious),
+                AdditionalFields = bag_pack(
+                        "threatUpdatedAt",
+                        threatInfo_updatedAt_t,
+                        "threatAnalystVerdict",
+                        threatInfo_analystVerdict_s,
+                        "threatIncidentStatus",
+                        threatInfo_incidentStatus_s,
+                        "mitigationStatus",
+                        mitigationStatus_s
+                    )
+            | project-rename
+                ThreatId = threatId_s,
+                ThreatName = threatInfo_threatName_s,
+                ThreatFirstReportedTime = threatInfo_identifiedAt_t,
+                ThreatCategory_threats = threatInfo_classification_s,
+                ThreatOriginalConfidence = threatInfo_confidenceLevel_s;
```

Comment:
is following info also available in log, can you pls check?
ThreatIsActive
ThreatIpAddr
ThreatField?

Reply:
Not available.